docs/data-guidelines.md

```diff
@@ -280,3 +280,26 @@ Members of this mixin are available to `HTMLAnchorElement` and `HTMLAreaElement`
    ```
 
 This guideline was proposed in [#8929](https://github.com/mdn/browser-compat-data/issues/8929), based in part on previous discussion in [#472](https://github.com/mdn/browser-compat-data/issues/472).
+
+## Choosing an experimental status
+
+Generally, when a feature is supported by one and only one browser engine, set `experimental` to `true`. When a feature is supported by two or more engines, then set `experimental` to `false`. Some exceptions apply, however, for long-standing features and features behind flags and prefixes.
+
+If a feature is supported behind flags only, no matter how many engines, then set `experimental` to `true`.
+
+If a feature is supported behind incompatible prefixes only (such as `-webkit-` in one engine and `-moz-` in another), no matter how many engines support the feature overall, then set `experimental` to `true`. If two or more engines support a feature behind a common prefix (such as `-webkit-` only), then set `experimental` to `false`.
+
+A single-engine feature's `experimental` status may expire and switch to `false` when the following conditions are met:
```

Comment:
OK, this picks up where @foolip left off at https://github.com/mdn/browser-compat-data/pull/9933#discussion_r616103899

> I would propose:
> 
> > Otherwise, if a feature has been supported by some browser for two or more years without major changes and there is no support behind a flag in some other browser with major differences from the first, then set `experimental` to `false`.

I thought this was a really clever way to sidestep my concern about intra-engine releases getting ahead (or behind) each other. I took this idea and rephrased it, because I think it's a little bit tough to parse as a single sentence.

What do you think?

Reply:
I like it, I don't have any nits, looks good as-is.